raiden/network/pathfinding.py

```diff
@@ -18,6 +19,31 @@
 log = structlog.get_logger(__name__)
 
 
+@unique
+class PfsError(IntEnum):
+    """ Error codes as returned by the PFS / defined in the PFS specs. """
+
+    INVALID_REQUEST = 2000
+    INVALID_SIGNATURE = 2001
+    REQUEST_OUTDATED = 2002
+
+    BAD_IOU = 2100
+    MISSING_IOU = 2101
+    WRONG_IOU_RECIPIENT = 2102
+    IOU_EXPIRED_TOO_EARLY = 2103
+    INSUFFICIENT_SERVICE_PAYMENT = 2104
+    IOU_ALREADY_CLAIMED = 2105
+    USE_THIS_IOU = 2106
+    DEPOSIT_TOO_LOW = 2107
```

Comment:
Could you either link to the spec page explaining those error codes? or maybe add docstrings explaining when each one could be returned?

Reply:
One more point, which is that fact that i don't see how/where we handle all of these errors? in `query_paths`, i can see we're handling some. So, either:
1. Remove the ones we don't handle if there is no need for them to be handled
2. Add additional error handling in `query_paths` if we should handle the ones we don't currently do in `query_paths`

Reply:
They are currently just defined as exception classes in the `raiden_services` repo. I added a reference to that in the docstring and a TODO to link to the PFS specs as soon as the error codes are documented there.

I'd like to leave all defined error codes in the enum as they are already all used in the unit tests.